content/pt-br/docs/ops/configuration/telemetry/in-proxy-service-telemetry/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Generate Istio Metrics Without Mixer [Alpha]
+description: How to enable in-proxy generation of HTTP service-level metrics.
+weight: 20
+aliases:
+  - /docs/ops/telemetry/in-proxy-service-telemetry
+---
+
+Istio 1.4 adds alpha support to generate service-level HTTP metrics
+directly in the Envoy proxies. This feature lets you continue to monitor your
+service meshes using the tools Istio provides without needing Mixer.
+
+The in-proxy generation of service-level metrics replaces the following HTTP
+metrics that Mixer currently generates:
+
+- `istio_requests_total`
+- `istio_request_duration_seconds`
+- `istio_request_size`
+
+## Enable service-level metrics generation in Envoy
```

Comment:
To fix the build errors, you'll need to change this section in the same way as https://github.com/istio/istio.io/pull/6308 because it's currently referencing files that have been deleted.